### PR TITLE
🌐 Add Chinese translation for `docs/advanced/response-headers.md`

### PR DESCRIPTION
**Pull request by jaystone776**
As title

### CODE REVIEW

File: docs/zh/docs/advanced/response-headers.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# 响应头
+
+## 使用 `Response` 参数
+
+与声明 Cookie 响应一样，*路径操作函数*中还可以声明 `Response` 类型的参数。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-headers.md:5]:
```suggestion
*路径操作函数*中可以声明 `Response` 类型的参数（正如声明响应的 cookie 时的做法一样）。
```

File: docs/zh/docs/advanced/response-headers.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+现在，在*临时*响应对象中设置响应头。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-headers.md:7]:
```suggestion
然后在*临时*的响应对象中设置响应头。
```

File: docs/zh/docs/advanced/response-headers.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    
+    **FastAPI** 的 `fastapi.responses` 与 `starlette.responses` 一样，只是为开发者提供的快捷方式，但其中绝大部分可用的响应都直接继承自 Starlette。
+    
+    `Response` 常用于设置响应头与 Cookie，因此，**FastAPI** 也在 `fastapi.Response` 中提供了支持。
+
+## 自定义响应头
+
+注意，自定义专用响应头可以添加<a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers" class="external-link" target="_blank">前缀 'X-'</a>。
+
+但如果需要浏览器中的客户端查看自定义响应头，则要在 CORS 配置中（详见 [CORS（跨域资源共享）](../tutorial/cors.md){.internal-link target=_blank})使用 <a href="https://www.starlette.io/middleware/#corsmiddleware" class="external-link" target="_blank">Starlette 官档 - CORS</a> 中的 `expose_headers` 参数。
```
Comment (waynerv) [inline docs/zh/docs/advanced/response-headers.md:43]:
```suggestion
但如果需要浏览器中的客户端查看自定义响应头，则要在 CORS 配置中（详见 [CORS（跨域资源共享）](../tutorial/cors.md){.internal-link target=_blank})使用 <a href="https://www.starlette.io/middleware/#corsmiddleware" class="external-link" target="_blank">Starlette 官方文档 - CORS</a> 中的 `expose_headers` 参数。
```

**Review by waynerv — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
It seems this was translated at some point in another PR. So I'll close this one, but thanks for the effort! ☕

**Pull request closed without merging**